addons/hr_holidays/__init__.py

```diff
@@ -6,3 +6,14 @@
 from . import populate
 from . import report
 from . import wizard
+
+from odoo import api, SUPERUSER_ID
+
+def _hr_holiday_post_init(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    french_companies = env['res.company'].filtered([('country_id.code', '=', 'FR')])
```

Comment:
This line will not do anything:
```suggestion
    french_companies = env['res.company'].search([('country_id.code', '=', 'FR')])
```